src/main/scala/rps/Main.scala

```diff
@@ -20,13 +20,21 @@ import model._
 import Result._
 import FailSupport._
 
+import rps.model.error._
+import rps.controller.{GameController}
+
 object Main extends App with RouterDerivationModule {
+  implicit def noGameResponse: ToHttpResponse[NoGameInMemory] =
+    (error: NoGameInMemory) =>
+      HttpResponse(
+        status = UnprocessableEntity,
```

Comment:
I didn't add an entity here because I had a Type mismatch that I couldn't solve... (maybe you could give me a hint @gabro  😄?)